V/VariantVisualization/Compat.toml

```diff
@@ -1,2 +1,5 @@
-[0]
+["0-0.3.5"]
 julia = "1"
+
+["0.3.6-0"]
+julia = "1.0.0-*"
```

Comment:
Maybe you can keep
```toml
[compat]
julia = "1.0"
```

Reply:
Thanks for the feedback. I'll make those changes. 

But why shouldn't I tag a release before registering? My understanding is that the TagBot would create a release but I'm not using that in favor of automated semantic release. That creates the tag, release, and changelog based on my commit messages, it also changes the version on my Project.toml, so I don't need to worry about version numbers or creating releases by hand.